src/mame/drivers/spg2xx.cpp

```diff
@@ -1066,6 +1066,81 @@ static INPUT_PORTS_START( hotwheels )
 	PORT_BIT( 0xffff, IP_ACTIVE_HIGH, IPT_UNUSED )
 INPUT_PORTS_END
 
+
+
+static INPUT_PORTS_START( doraphone )
+	PORT_START("P1")
+	PORT_CONFNAME( 0x0070, 0x0060, "On/Off Mode Slider" )
+	PORT_CONFSETTING(      0x0030, "Play Alone (no video)" )
+	PORT_CONFSETTING(      0x0060, "Play on TV" )
+
+	PORT_BIT( 0x0080, IP_ACTIVE_HIGH, IPT_BUTTON2 ) PORT_PLAYER(1) PORT_NAME("Lift Handset") // this could be done as a toggle, although note, handset being down is treated like a button being held
+	PORT_BIT( 0x0100, IP_ACTIVE_HIGH, IPT_UNUSED ) // this setting is US NTSC, ACTIVE_LOW gives US PAL (invalid?) no way to switch to non-US?
```

Comment:
If it has an effect but needs a specific sense value, it’s better to use `IPT_CUSTOM` or something rather than `IPT_UNUSED` (e.g. see the sense lines for AES/MVS, MVS/JAMMA and 4/6 slot in neogeo.cpp).